README.md

```diff
@@ -25,77 +35,84 @@ Add this dependency to your project's POM:
 
 ## Usage for DSPs
 
-See com.uid2.client.test.IntegrationExamples for example usage for DSPs.
+For example usage for DSPs, see com.uid2.client.test.IntegrationExamples.**(gwh/JN_01 where is this? Can we link?)**
 
 ## Usage for Publishers
 
 As a publisher, there are two ways to use this SDK: 
-1. **Basic usage** is for those publishers that are happy to use this SDK's HTTP implementation (synchronous [OkHttp](https://square.github.io/okhttp/)).
-2. **Advanced usage** is for those publishers that prefer to use their own HTTP library. 
+1. [**Basic Usage**](#basic-usage) is for publishers who are happy to use this SDK's HTTP implementation (synchronous [OkHttp](https://square.github.io/okhttp/)).
+2. [**Advanced Usage**](#advanced-usage) is for publishers who prefer to use their own HTTP library. 
+
+For an example application that demonstrates both Basic and Advanced usage, see [uid2-examples](https://github.com/UnifiedID2/uid2-examples/tree/main/publisher/uid2-java-test-site).
 
+### Basic Usage
 
-An example application which demonstrates both Basic and Advanced usage is at [uid2-examples](https://github.com/UnifiedID2/uid2-examples/tree/main/publisher/uid2-java-test-site).
+If you're using the SDK's HTTP implementation, follow these steps.
 
-### Basic usage
 1. Create an instance of PublisherUid2Client as an instance variable.
  
    `private final PublisherUid2Client publisherUid2Client = new PublisherUid2Client(UID2_BASE_URL, UID2_API_KEY, UID2_SECRET_KEY);`
 
-2. When a user authenticates and authorizes the creation of a UID2:
+2. When a user authenticates and authorizes the creation of a UID2: **(gwh/JN_02 action? Does the user have to do something here?)**
```

Comment:
This is step 1-b and 1-c here: https://github.com/IABTechLab/uid2docs/blob/main/api/v2/guides/custom-publisher-integration.md

Reply:
@jon8787 : Thx but I wasn't clear. What I meant was, what is our reader doing in this step? A statement of what "IdentityTokens identity = publisherUid2Client.generateToken(TokenGenerateInput.fromEmail(emailAddress));" does.